src/deploydocs.jl

```diff
@@ -379,7 +379,7 @@ function git_push(
     # Generate a closure with common commands for ssh and https
     function git_commands(sshconfig = nothing)
         # Setup git.
-        run(`$(git()) init`)
+        run(`$(git()) init --initial-branch=master`) # specify initial branch just to turn off the advice in the deploy logs
```

Comment:
Off topic, but why "master", hasn't the default been "main" for several years now?